components/core/core.module.ts

```diff
@@ -36,6 +35,7 @@ import {HsToolbarModule} from '../toolbar/toolbar.module';
 import {HsUtilsModule} from './../utils';
 import {HttpClientModule} from '@angular/common/http';
 import {NgModule} from '@angular/core';
+import {TranslateModule} from '@ngx-translate/core';
```

Comment:
Not sure why but  my webpack throws error with this import. Even in my branch https://github.com/hslayers/hslayers-ng/blob/ngx-translate/components/core/core.module.ts I had to use full path. Does it work for you @jmacura ?

Reply:
Was talking to Dailis about this one and it seems like error on my side...nevertheless according to docs it is recommended to use  12.x+ version of ngx-translate/core with Angular 9.

Reply:
> 
> 
> Not sure why but my webpack throws error with this import. Even in my branch https://github.com/hslayers/hslayers-ng/blob/ngx-translate/components/core/core.module.ts I had to use full path. Does it work for you @jmacura ?

Nice and clean. No error anywhere.